src/main/java/com/hazelcast/remotecontroller/CloudManager.java

```diff
@@ -0,0 +1,428 @@
+package com.hazelcast.remotecontroller;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.*;
+import okhttp3.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import net.lingala.zip4j.ZipFile;
+import org.python.jline.internal.Log;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.*;
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public class CloudManager {
+    private static final OkHttpClient client = new OkHttpClient();
+    private String baseUrl;
+    private final ObjectMapper mapper = new ObjectMapper();
+    private String bearerToken;
+    private static final Logger LOG = LogManager.getLogger(Main.class);
+    private static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
+    private Call call;
+    private static final int timeoutForClusterStateWait = 5;
+    private static final int retryTimeInSecond = 10;
+    private static final int clusterTypeId = 5; // Serverless cluster
```

Comment:
Where are these constants defined? Can they change without us noticing it?

Reply:
These constants are returned by the API https://api.dev.viridian.hazelcast.cloud/swagger-ui/index.html?configUrl=%2Fswagger-resources%2Fswagger-config&urls.primaryName=Orchestrator

Actually the whole `dev` environment will be replaced with `dev1` and `dev2` as I learned. The constants may change as well. So we will need to update some things for sure.

Reply:
OK, I hope things will be stabilized when the viridian hits GA.